scarb-agent/scarb-agent-new/templates/run-service.hbs

```diff
@@ -15,12 +15,12 @@ spec:
         - image: "europe-west1-docker.pkg.dev/giza-platform-common/platform-images/orion-runner:0.2.6preb44771"
           name: agent-runner
           ports:
-            - containerPort: 8080
+            - containerPort: 3000
           env:
-          - name: REGISTRY_URL
-            value: "%REGISTRY_PATH%"
-          - name: PREPROCESS_URL
-            value: 'http://localhost:8888/preprocess'
```

Comment:
`containerPort` refers to Orion Runner which starts up at `8080` by default. If you want to change python server one it should be done on the `PREPROCESS_URL`